vhdbuilder/scripts/cni-prefetch-driver.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+CNI_PREFETCH_SCRIPT_PATH="/opt/azure/containers/prefetch.sh"
```

Comment:
lets rename everything to not specifically reference CNI since we're gonna try to generalize according to Will's feedback